### DISCUSSION TITLE
how delete a dmca repository?

### DISCUSSION THREAD

**Discussion by iOSPrincekin** [community/community#58993] [Repositories]
### Select Topic Area

General

### Body

hi:
     how to delete a dmca repository? can you help me delete it, thanks a lot!

**Comment by ikozigbo** [comment]
If you have received a DMCA takedown notice for a repository on GitHub and you wish to remove it, here are the general steps you can follow:

1. Review the DMCA notice: Read the DMCA takedown notice carefully to understand the specific allegations and which repository it pertains to.

2. Assess the validity of the notice: Evaluate the notice to determine its legitimacy. If you believe it is valid and the repository does indeed contain copyrighted material without authorization or violates your rights, proceed with the next steps.

3. Contact the repository owner: Reach out to the owner of the repository and inform them about the DMCA notice you've received. You can explain the situation and request that they voluntarily remove the infringing content.

4. Provide a timeframe: Specify a reasonable deadline for the repository owner to remove the infringing content, usually within 24 to 48 hours. Make sure to document your communication with the repository owner.

5. Submit a DMCA takedown request to GitHub: If the repository owner doesn't respond or refuses to remove the infringing content, you can proceed with submitting a formal DMCA takedown request to GitHub. Follow the steps I mentioned in my previous response to prepare and submit the takedown notice.

6. Await GitHub's response: GitHub will review your DMCA takedown request and take appropriate action. They may require additional information or clarification during the process. GitHub will communicate with you regarding the status of your request.

It's important to note that the specific process may vary depending on the circumstances and the policies of GitHub. If you're unsure about any steps or legal aspects of the process, it's recommended to consult GitHub's official documentation or seek legal advice for accurate and up-to-date information on how to handle DMCA takedown requests.

**Reply by queenofcorgis** [reply]
Hey @iOSPrincekin - just out of curiosity did you use ChatGPT or another similar LLM to write this answer? We're still working through how we want to include ChatGPT in the community so asking our users helps us learn more!

**Comment by iOSPrincekin** [comment]
I am the owner of the [repository](https://github.com/iOSPrincekin/iBoot) repository, this repository had been disabled, how to delete it, thanks!

**Comment by airtower-luna** [comment]
You'll have to [contact GitHub support](https://support.github.com/request) about that.
